attendance/views.py

```diff
@@ -388,7 +389,7 @@ def getAttendance(request, pk):
             return JsonResponse(result, safe=False)
 
     query_class = SubjectClass.objects.get(pk=pk)
-    response = fetchLatestAttendances(query_class)
+    response = fetchLatestAttendances(request,query_class)
```

Comment:
in this function you should update can_mark_attendance property in line below `result=cache.get(cache_key)` and before returning response